### DISCUSSION TITLE
Trying to compute the expectation of a multivariate normal

### DISCUSSION THREAD

**Discussion by AdrienCorenflos** [sympy/sympy#22777] [Q&A]
Hello, 

I am trying to compute an integral of a multivariate Gaussian but I don't seem to be able to understand how to feed it to SymPy properly...

This is what I am trying to compute (trying to get a closed-form solution)

```python
from sympy.stats import E, MultivariateNormal, density
from sympy import symbols, MatrixSymbol, minimum

x = symbols("x", real=True)
n = 3

P = MatrixSymbol('P', n, n)
Q = MatrixSymbol('Q', n, n)

m = MatrixSymbol('m', n, 1)
mu = MatrixSymbol('mu', n, 1)

X = MultivariateNormal('X', m, P)
Y = MultivariateNormal('Y', m, Q)
Z = MultivariateNormal('Z', mu, Q)

# I want to compute E[min(1, N(X; mu, Q) / N(X; m, Q))] for X~N(m, P)
E(minimum(density(Z)(X) / density(Y)(X) , 1))  

```

**Comment by oscargus** [comment]
You probably want to use `Min` instead of `minimum`. Not sure it will evaluate to something useful.

(`minimum` is computing the min of a function, in this case subject to the variable 1.)

**Comment by oscargus** [comment]
No, that wasn't the problem (yet, just answered on the phone without running your code...).

The problem right now is that the arguments of `density(Z)` should be variables not distributions. (Which may or may not lead you to what you actually need, not sure how to compute that.)
